script.service.kodi.callbacks/resources/lib/utils/poutil.py

```diff
@@ -304,38 +608,115 @@ def getFileList(self):
         return files_to_scan
 
     def scanPyFilesForStrings(self):
+        """
+        Scans all .py files in directory tree for localized strings
+        :return: List of strings needing localization
+        :rtype: list
+        """
         files = self.getFileList()
         lstrings = []
         for myfile in files:
-            finds = []
-            with open(myfile, 'r') as f:
-                lines = ''.join(f.readlines())
+            with codecs.open(myfile, 'r', 'utf-8') as f:
+                lines = u''.join(f.readlines())
             try:
                 finds = self.find_localizer.findall(lines)
             except re.error:
-                pass
+                finds = []
             finally:
                 if len(finds) != 1:
-                    log(msg='Skipping file: %s, localizer not found' % myfile)
+                    log(msg=u'Skipping file: %s, localizer not found' % myfile)
                 else:
-                    findstr = r"%s\('(.+?)'\)" % finds[0]
+                    findstr = ur"%s\('(.+?)'\)" % finds[0]
                     find = re.compile(findstr)
-                    finds = []
                     try:
                         finds = find.findall(lines)
                     except re.error:
-                        pass
+                        finds = []
                     lstrings += finds
         return lstrings
 
```

Comment:
Not sure what this is used for, but might be worth havin a look at https://pypi.python.org/pypi/polib